fastlane/Fastfile

```diff
@@ -258,14 +258,17 @@ end
 
 platform :android do
   # Optional env variables
-  APK_PATH = ENV['APK_PATH'] || 'result/app.apk'
+  APK_PATHS = ["result/app.apk"]
```

Comment:
```suggestion
  APK_PATHS = ENV["APK_PATHS"]&.split(";") or ["result/app.apk"]
```

Reply:
Oooh, I never heard about this: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/